core/src/autogluon/core/utils/try_import.py

```diff
@@ -110,3 +111,11 @@ def try_import_gluonnlp():
             "without installing gluonnlp. "
             "A quick tip is to install via `pip install gluonnlp==0.8.1`. ")
     return gluonnlp
+
+
+def try_import_torch():
+    try:
+        import torch
+    except ImportError as e:
+        raise ImportError("Unable to import dependency torch\n"
+                          "A quick tip is to install via `pip install torch`.")
```

Comment:
Do you know if TT has specific torch version requirements?

Reply:
It looks like Josh, et al. were using a slightly earlier version of torch since I had to add the code for removing periods from columns.
I could do some testing to see what it would do with a version or two back. I'm currently just running on latest.

Reply:
no worries, we can just specify min version once a user reports a bug